config/hardware/accelerometers/adxl345_btt_lis2dw.cfg

```
@@ -0,0 +1,26 @@
+# This ADXL file is dedicated to be used with ADXL boards
+# connected over USB to the pi as dedicated and standalone ADXL-MCU boards
+
+# https://github.com/bigtreetech/LIS2DW
+
+# You need to override the following to be able to set the proper serial in your overrides.cfg file
+[mcu adxl]
+serial: /dev/serial/by-id/xxx
+
+[adxl345]
+cs_pin: adxl:gpio9
+#spi_bus: spi1a
+spi_software_sclk_pin: adxl:gpio10
+spi_software_mosi_pin: adxl:gpio11
+spi_software_miso_pin: adxl:gpio8
+axes_map: -y,x,-z
+
+[resonance_tester]
+probe_points: -1,-1,-1
+accel_chip: adxl345
```

Same here since the name of the section is different, it need also to be changed here

```suggestion
accel_chip: lis2dw
```